src/Features/Lsif/Generator/Graph/DefinitionRangeTag.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using LSP = Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator.Graph
+{
+    internal class DefinitionRangeTag : RangeTag
+    {
+        public DefinitionRangeTag(string text, LSP.SymbolKind symbolKind, LSP.Range fullRange) : base(type: "definition", text)
+        {
+            SymbolKind = symbolKind;
```

Comment:
I think this is supposed to be 'kind' rather than 'symbolKind'

```typescript
/**
 * The range represents a definition
 */
export interface DefinitionTag {
  /**
   * A type identifier for the declaration tag.
   */
  type: 'definition';

  /**
   * The text covered by the range
   */
  text: string;

  /**
   * The symbol kind.
   */
  kind: lsp.[SymbolKind](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#symbolKind);

  /**
   * The full range of the definition not including leading/trailing whitespace
   * but everything else, e.g comments and code. The range must be included in
   * fullRange.
   */
  fullRange: lsp.[Range](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#range);

  /**
   * Optional detail information for the definition.
   */
  detail?: string;
}
```

Reply:
Good catch! Will fix.